libs/error-stack/.justfile

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env just --justfile
+
+set fallback
+
+[private]
+@default:
+  just usage
+
+# TODO: add `--ignore-unknown-features` to `cargo hack` and pass `--workspace`
+cargo-hack-groups := '--group-features eyre,hooks --group-features anyhow,serde'
+profile := env_var_or_default('PROFILE', "dev")
+
+[private]
+clippy *arguments:
+  @just install-cargo-hack
+
+  @just in-pr cargo clippy --profile {{profile}} --workspace --all-features --all-targets --no-deps {{arguments}}
+  @just not-in-pr cargo hack --optional-deps --feature-powerset {{cargo-hack-groups}} --ignore-unknown-features clippy --profile {{profile}} --all-targets --no-deps {{arguments}}
+
+[private]
+test *arguments:
```

Comment:
Any reason why you don't have the other syntax for these?
```suggestion
@test *arguments:
```

Reply:
`@` in front of the job hides all `stderr` output emitted by `just`, so we don't need to apply it to every line. However, I'm fine avoiding this syntax.

Reply:
I preferred `@` per line as this was more explicit but don't really have strong feelings.

Reply:
I don't feel strongly either, but we're inconsistent at the moment, that's what I'm highlighting.

Even for jobs where we're applying `@` to every line, we aren't using it on the top level command. But then on some other ones we do